### DISCUSSION TITLE
Noise level and time scaling for signal level plot.

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#96] [Ideas]
It would be very useful to have the noise level (noise floor) and the time scaling on the signal level graphs.

Sometimes this happens, but it is not clear why.
![изображение](https://user-images.githubusercontent.com/20313019/228810172-f025db27-e3d6-4b07-80b5-19863674d193.png)
![изображение](https://user-images.githubusercontent.com/20313019/228809672-0179feda-f509-4908-b3da-17ca5bf27796.png)
Example from Graphs1090:
![изображение](https://user-images.githubusercontent.com/20313019/228839794-9837f471-f68b-4e7d-a214-a383f60e9e7e.png)

It would also be excellent if all the graphs could be scaled at the same time. (showed equal intervals of time). How it is done in Graphs1090.

Thank you very much for your efforts!
MaxPi

**Comment by jvde-github** [comment]
Hey, thanks. What do you mean with lacking, don't understand the relation between the plots and signal level comments.

**Reply by Taxom** [reply]
I edited my comment above. Sorry for my English :))
